_harp/academy/_faq.md

```diff
@@ -22,7 +22,10 @@ Courses are peer-led and project-based with sixteen students working in four tea
 Primarily, we cover *test-driven* development, using *JavaScript* and *Node.js*. We also cover aspects of UX design, product management and entrepreneurship.
 
 **What projects will I work on in the second part of the programme?**  
-We aim to give you an opportunity to work on a variety of open source, charitable, social enterprise, startup and commercial projects. The programme is under continuous development and the balance between learning and earning a living is always under review.  
+We aim to give you an opportunity to work on a variety of open source, charitable and social enterprise projects. The programme is under continuous development and the balance between learning and creating useful software is always under review.  
+
+**Will I be able to earn a living working on projects?**  
+After you complete your 16 weeks, we will aim to find you commercial work as soon as we can, either on a freelance basis or with an employer.
 
```

Comment:
I would just reword it, something like:
After you complete your 16 weeks, we will usually have some people contacting us for commercial MVPs which we will pass on to you as and when they come in. If your aim is to find a job as a junior developer with a full time employer, we will do what we can to help you find one.